vivisect/extensions/__init__.py

```diff
@@ -50,7 +50,7 @@ def loadExtensions(vw, vwgui):
                     filebytes = fd.read()
                 exec(filebytes, module.__dict__)
                 module.vivExtension(vw, vwgui)
-                vw.addExtension(modpath, module)
+                vw.addExtension(fname, module)
```

Comment:
fname is fine here... but what *must* change is the "open(fname..." which needs to become "open(modpath..."

Reply:
:+1: